corehq/apps/userreports/util.py

```diff
@@ -22,3 +27,11 @@ def localize(value, lang):
 
 def default_language():
     return "en"
+
+
+def validate_sql_column_name(s):
+    if not isinstance(s, unicode):
+        s = unicode(s, "utf-8")
+    # http://stackoverflow.com/questions/954884/what-special-characters-are-allowed-in-t-sql-column-name
```

Comment:
noticed this question is specific to microsoft SQL. did you find any resources for postgres?